docs/sources/next/javascript-api/k6-experimental/browser/jshandle/evaluatehandle.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'evaluateHandle(pageFunction[, arg])'
+description: 'Browser module: JSHandle.evaluateHandle(pageFunction[, arg]) method'
+---
+
+# evaluateHandle(pageFunction[, arg])
+
+Evaluates the page function and returns a `JSHandle`. This method passes this handle as the first argument to the page function. Unlike [evaluate](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-experimental/browser/jshandle/evaluate/), `evaluateHandle` returns the value as a `JSHandle`
```

Comment:
We used the code block formatting for a "page function" [here](https://github.com/grafana/k6-docs/pull/1617/commits/8ccd6264edcd99026c7d4dace7030b4d6d51af9a). Should we be consistent?

Reply:
Yep, good spot.

Changed in aa85fa243c90067ab6532dca6af2a6f596901109